README.md

```diff
@@ -46,6 +46,7 @@ Debian system.
 ##### Install (upgrade) required Python libraries
 
 	$ sudo pip install --upgrade trappy bart-py devlib
+	$ sudo pip install netifaces
```

Comment:
Yes, it can be installed via apt-get, but since it is a python package isn't it more reliable to install it via pip?